app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,292 @@
+    </string-array>
+    <string-array name="url_array">
+        <item>Visita URL</item>
+        <item>Non fare niente</item>
+    </string-array>
+    <string-array name="product_array">
+        <item>Cerca sul Web</item>
+        <item>Non fare niente</item>
+    </string-array>
+    <string-array name="wifi_array">
+        <item>Copia Password</item>
+        <item>Non fare niente</item>
+    </string-array>
+    <string-array name="history_array">
+        <item>Copia nella clipboard</item>
+        <item>Cancella valore</item>
+    </string-array>
+    <string-array name="generator_array">
+        <item>E-mail</item>
+        <item>URL</item>
+        <item>Numero di telefono</item>
+        <item>SMS</item>
+        <item>Geolocalizzazione</item>
+        <item>MeCard</item>
+        <item>BizCard</item>
+        <item>MMS</item>
+        <item>Wifi</item>
+        <item>VCard</item>
+        <item>Market</item>
+
+    </string-array>
+    <string name="copied_to_clipboard">Copiato nella clipboard.</string>
+    <string name="noname">Nessun nome specificato</string>
+
+    <string name="title_activity_result_url">Collegamento trovato</string>
+    <string name="title_activity_result_text">Testo trovato</string>
+    <string name="title_activity_result_sms">SMS trovato</string>
+    <string name="title_activity_result_email">Indirizzo E-mail trovato</string>
+    <string name="title_activity_result_send_email">E-mail trovata</string>
+    <string name="title_activity_result_product">Prodotto trovato</string>
+    <string name="title_activity_result_tel">Numero di telefono trovato</string>
+    <string name="title_activity_result_wifi">WiFi trovata</string>
+    <string name="title_activity_result_contact">Contatto trovato</string>
+    <string name="action_help">Aiuto</string>
+
+    <!-- ### WELCOME DIALOG ### -->
+    <string name="welcome_text">L\'app PRIVACY FRIENDLY QR Code Scanner consente di scansionare codici a barre e codici-QR di varia tipologia richiedendo un numero minimo di permessi e rilevando collegamenti dannosi.</string>
+    <string name="welcome">Benvenuto</string>
+    <string name="viewhelp">Visualizza Aiuto</string>
+
+    <!--<string name="help_privacy_heading">Privacy Info</string>
+    <string name="about_author_contributors">Contributors</string>
+    <string name="privacy_friendly">This application belongs to the group of PRIVACY FRIENDLY Apps developed by Technische Universität Darmstadt. Sourcecode licensed under GPLv3. Images copyright TU Darmstadt and Google Inc.</string>
+    <string name="more_info">More information can be found on:</string>
+    <string name="github" translatable="false"><a href="https://github.com/SecUSo/privacy-friendly-qr-scanner">Github-Repo</a></string>-->
+    <string name="help_permission1_description">L\'applicazione richiede il permesso di accedere alla fotocamera, che viene utilizzato per scansionare i codici a barre ed i codici QR attraverso la fotocamera.</string>
```

Comment:
che viene utilizzata

Reply:
"che viene utilizzato", we are talking about the **permission**, hence, "utilizzato"

Do you agree?

Reply:
```L\'applicazione richiede il permesso di accedere alla fotocamera, utilizzata per scansionare i codici a barre ed i codici QR.```